### PR TITLE
Add a subsection on prefetching to the documentation of next/Link

### PR DESCRIPTION
**Pull request by zmilonas**
Inspired by https://github.com/vercel/next.js/discussions/11793#discussioncomment-267024

Probably could use a little more content but I couldn't think of much more at the moment

### CODE REVIEW

**Comment by leerob** [comment]
Thanks for contributing! However, this is the same content as above, and could be confusing why it's duplicated.

**Comment by zmilonas** [comment]
Thanks for taking the time to review @leerob.

As to duplication, yeah I was not sure where to put the content so I decided to add another section the gist of what I wanted to add is the "with `preload=false` there is still preload happening on hover" to underline that it's actually not entirely turned off with `preload=false`.

I don't see that being anywhere on that page yet and I think it'd be nice to spin off the preload section and describe the behavior.

File: docs/api-reference/next/link.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default Home
...
 - [`passHref`](#if-the-child-is-a-custom-component-that-wraps-an-a-tag) - Forces `Link` to send the `href` property to its child. Defaults to `false`
-- `prefetch` - Prefetch the page in the background. Defaults to `true`. Any `<Link />` that is in the viewport (initially or through scroll) will be preloaded. Prefetch can be disabled by passing `prefetch={false}`. Pages using [Static Generation](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) will preload `JSON` files with the data for faster page transitions
+- [`prefetch`](#prefetching) - Prefetch the page in the background. Defaults to `true`. Any `<Link />` that is in the viewport (initially or through scroll) will be preloaded. Prefetch can be disabled by passing `prefetch={false}`. Pages using [Static Generation](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) will preload `JSON` files with the data for faster page transitions
```
Comment (leerob) [inline docs/api-reference/next/link.md:60]:
```suggestion
- `prefetch` - Prefetch the page in the background. Defaults to `true`. Any `<Link />` that is in the viewport (initially or through scroll) or hovered will be preloaded. Prefetch can be disabled by passing `prefetch={false}`. Pages using [Static Generation](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) will preload `JSON` files with the data for faster page transitions
```

File: docs/api-reference/next/link.md [hunk 2]
```diff
@@ -208,3 +208,7 @@ The default behavior of `Link` is to scroll to the top of the page. When there i
   <a>Disables scrolling to the top</a>
 </Link>
 ```
+
+## Prefetching
+
+The default behavior of `Link` is to preload any link that is in the viewport. With `prefetch={false}` the links will be preloaded on hover.
```
Comment (leerob) [inline docs/api-reference/next/link.md:214]:
```suggestion
```

**Comment by leerob** [comment]
Related https://github.com/vercel/next.js/pull/18859

**Comment by zmilonas** [comment]
Ok I guess that works as well, thanks @leerob

**Review by leerob — APPROVED** [review]

**Comment by leerob** [comment]
Oh, this is in prod now.

`When `prefetch` is set to `false`, prefetching will still occur on hover.`

**Pull request closed without merging**